### DISCUSSION TITLE
Flat config for Vue

### DISCUSSION THREAD

**Discussion by bmulholland** [graphql-hive/graphql-eslint#2339] [Q&A] [answered]
The example for vue (https://github.com/dimaMachina/graphql-eslint/tree/master/examples/vue-code-file) hasn't been upgraded to flat config, and I'm trying to figure it out. Currently, I get `GraphQL template literals cannot be plucked from a Vue template code without having the "@vue/compiler-sfc" package installed.`

I do have that package installed, though. Version 3.4.27.

Maybe this is why that example isn't ported? I don't see a relevant Issue describing that this isn't expected to work though. Is it possible to get working right now? Is there an Issue to follow that describes what's needed to add support? If it is possible now, could I get some help getting it working?

The relevant config looks like this:
```
  {
    // Setup GraphQL Parser
    files: ["**/*.{graphql,gql}"],
    plugins: {
      "@graphql-eslint": graphqlESLint,
    },
    languageOptions: {
      parser: graphqlESLint,
    },
  },
```

**Comment by dimaMachina** [comment, marked as answer]
looks like it is fixed, thanks for [your contribution ](https://the-guild.dev/graphql/eslint/docs/usage/vue)
